core/server/master/src/main/java/alluxio/master/file/meta/LockedInodePath.java

```diff
@@ -81,6 +85,42 @@ public class LockedInodePath implements Closeable {
   @Nullable
   private final ResourceLeakTracker<LockedInodePath> mTracker;
 
+  /**
+   * Keeps a reference of JournalContext and flushes it before the lock is release.
+   * This is used to prevent inconsistency between primary and standby.
+   * A typical write operation in FileSystem looks like:
+   * try (createJournalContext()) {
```

Comment:
From the other parts of the codebase i see people often put HTML tags in the java doc as below
```suggestion
   * <blockquote><pre>
   * try (createJournalContext()) {
   * ...
   * </pre></blockquote>
```